components/server/src/initialization/database.py

```diff
@@ -44,6 +46,14 @@ def init_database() -> Database:  # pragma: no cover-behave
     return database
 
 
+def set_feature_compatibility_version(database: Database) -> None:
```

Comment:
```suggestion
def set_feature_compatibility_version(admin_database: Database) -> None:
```

To indicate that this not the same database as all the other database arguments that we have...